### PR TITLE
add honest greens

### PR DESCRIPTION
**Pull request by AlertSubject**
closes: https://github.com/osmlab/name-suggestion-index/issues/7850

### CODE REVIEW

File: data/brands/amenity/restaurant.json [hunk 1]
```diff
@@ -2177,6 +2177,17 @@
         "name": "Honest Burgers"
       }
     },
+    {
+      "displayName": "Honest Greens",
+      "id": "honestgreens-acf031",
+      "locationSet": {"include": ["es"]},
```
Comment (Cj-Malone) [inline data/brands/amenity/restaurant.json:2183]:
```suggestion
      "locationSet": {"include": ["es", "pt"]},
```

@codeinabox mentioned Portugal.

**Comment by codeinabox** [comment]
I'm not sure I'd tag this as a restaurant as you have to order at the counter and there is no table service

**Comment by AlertSubject** [comment]
> I'm not sure I'd tag this as a restaurant as you have to order at the counter and there is no table service

They themselves call it restaurants so I went with that
https://honestgreens.com/en/our-restaurants-2/

It seems to not entirely match up with the description of restaurants that can be found in the osm Wiki
Fastfood seems more fitting maybe?
https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dfast_food

Is there no table service or no tables?

If tables are present then I think restaurant could still be a fitting tag even if there's no service

**Comment by UKChris-osm** [comment]
The POI's currently in OSM seem to all be marked as `amenity=restaurant`, if that makes any difference?

**Comment by codeinabox** [comment]
> The POI's currently in OSM seem to all be marked as `amenity=restaurant`, if that makes any difference?

In that case let's go with restaurant, though I think cafe would be equally as appropriate as they serve many breakfast items and you order at the counter and they bring it to your table

**Comment by tas50** [comment]
Sounds like a fast casual restaurant which would be restaurant in OSM

**Pull request merged**